static/app/components/events/featureFlags/eventFeatureFlagList.spec.tsx

```diff
@@ -208,8 +217,25 @@ describe('EventFeatureFlagList', function () {
     ).toBeInTheDocument();
   });
 
-  it('renders nothing if event.contexts.flags is not set', function () {
-    render(<EventFeatureFlagList {...NO_FLAG_CONTEXT_SECTION_PROPS} />);
+  it('renders cta if event.contexts.flags is not set and should show cta', async function () {
+    render(<EventFeatureFlagList {...NO_FLAG_CONTEXT_SECTION_PROPS_CTA} />);
+    screen.logTestingPlaygroundURL();
```

Comment:
debug?

Reply:
oops, good catch. left that in there accidentally